src/bytes.ts

```diff
@@ -112,7 +112,7 @@ export const unpadHexString = function(a: string): string {
  */
 export const toBuffer = function(
   v:
-    | string
+    | PrefixedHexString
```

Comment:
I'm assuming that originally we expected these strings to be `0x` prefixed?

Reply:
yeah actually `PrefixedHexString` is simply a string type right now, maybe in the future typescript will give us some way to enforce that the first two characters are `0x`.